server/src/main/java/com/battleships/Messages/LogMessages.java

```diff
@@ -0,0 +1,13 @@
+package com.battleships.Messages;
+
+public class LogMessages {
```

Comment:
consider using enums

Reply:
We prefer static Strings instead of making trains like LogMessages.DISCONNECTED_AFTER_PLAYER_REQ.toString()

If you have better idea please let us know